packages/wordcount/src/index.js

```diff
@@ -0,0 +1,102 @@
+import { extend, flow  } from 'lodash';
+import { defaultSettings } from './defaultSettings'
+import stripTags from './stripTags';
+import transposeAstralsToCountableChar from './transposeAstralsToCountableChar';
+import stripHTMLEntities from './stripHTMLEntities';
+import stripConnectors from './stripConnectors';
+import stripRemovables from './stripRemovables';
+import stripHTMLComments from './stripHTMLComments';
+import stripShortcodes from './stripShortcodes';
+import stripSpaces from './stripSpaces';
+import transposeHTMLEntitiesToCountableChars from './transposeHTMLEntitiesToCountableChars';
+
```

Comment:
I belive multiple empty lines are not allowed in our eslint rules (which will be integrated to this repository soon I guess)